sdk/tables/Azure.Data.Tables/readme.md

```diff
@@ -3,22 +3,22 @@
 Azure Table storage is a service that stores large amounts of structured NoSQL data in the cloud, providing 
 a key/attribute store with a schema-less design. 
 
-Azure Cosmos DB provides a Table API for applications that are written for Azure Table storage and that need premium capabilities like:
+Azure Cosmos DB provides a Table API for applications that are written for Azure Table storage that need premium capabilities like:
 
 - Turnkey global distribution.
 - Dedicated throughput worldwide.
 - Single-digit millisecond latencies at the 99th percentile.
 - Guaranteed high availability.
 - Automatic secondary indexing.
 
-The Azure Tables client library can seamlessly target either Azure table storage or Azure Cosmos DB table service endpoints with no code changes.
+The Azure Tables client library can seamlessly target either Azure Table storage or Azure Cosmos DB table service endpoints with no code changes.
 
 [Source code][table_client_src] | [Package (NuGet)][table_client_nuget_package] | [API reference documentation][api_reference] | [Samples][table_client_samples]
 
 ## Getting started
 
 ### Install the package
-Install the Azure Tables client library for .NET with [NuGet][nuget]:
+Install the Azure Tables client library for .NET with [NuGet][table_client_nuget_package]:
```

Comment:
Should this link to the future page? https://www.nuget.org/packages/Azure.Data.Tables

Reply:
The CI build's link validation won't allow it until that url resolves. I was planning to update it once we release next week.